src/style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  margin: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.FirstForm {
+  margin: 20px 8px -3px;
+}
+
+.search-1 {
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
+  padding-left: 62px;
+  padding-bottom: 2px;
+
+  background: #fff;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.BigLoupe {
+  position: relative;
+  top: -45px;
+  left: 26px;
+}
+
+.SecondForm {
+  margin: 0 8px;
+}
+
+.search-2 {
+  width: 100%;
+  margin: 0;
+  height: 42px;
+  box-sizing: border-box;
+  padding-left: 33px;
+  padding-bottom: 2px;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.SmallLoupe {
+  position: relative;
+  top: -31px;
+  left: 13px;
+  width: 11px;
+}
+
+.search::placeholder {
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: "Avenir-Heavy", sans-serif;
```

Comment:
if you connect your `fonts` correctly - you wouldn't need this, only change font-weight on `focus`